src/cosmosdb-preview/azext_cosmosdb_preview/_validators.py

```diff
@@ -170,6 +171,60 @@ def _parse_resource_path(resource,
     return result['resource_id']
 
 
+def validate_gossip_certificates(ns):
+    """ Extracts multiple comma-separated certificates """
+    if ns.external_gossip_certificates is not None:
+        ns.external_gossip_certificates = get_certificates(ns.external_gossip_certificates)
+
+
+def validate_client_certificates(ns):
+    """ Extracts multiple comma-separated certificates """
+    if ns.client_certificates is not None:
+        ns.client_certificates = get_certificates(ns.client_certificates)
+
+
+def get_certificates(input_certificates):
+    from azext_cosmosdb_preview.vendored_sdks.azure_mgmt_cosmosdb.models import Certificate
+    certificates = []
+    for item in input_certificates:
+        for i in item.split(","):
```

Comment:
I'm curious since you have defined the param as `nargs=+`, why would users use ',' to separate? This might expect the input to be `--client-certificates cer1 cer2,cer3 cert4`, right?

Reply:
fixed.